.github/repo_meta.yaml

```diff
@@ -0,0 +1,31 @@
+# point_of_contact: the owner of this repository, can be a GitHub user or GitHub team
+point_of_contact: sfc-gh-mrui sfc-gh-bli sfc-gh-ema sfc-gh-sshankar
```

Comment:
Hi, Nathan, Can we add multiple user as `point_of_contact`? If so, how to do that?